Modules/Microsoft365DSC/DSCResources/MSFT_EXOMailboxIRMAccess/settings.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+    "resourceName": "EXOMailboxIRMAccess",
+    "description": "Description of what the resource is about.",
+    "roles": {
+        "read": [
+            "Role"
```

Comment:
@PiyushDub we need to update this file. This is what is currently causing the Unit tests to fail.